fastn-core/src/auth/email_password/login.rs

```diff
@@ -24,10 +25,37 @@ pub(crate) async fn login(
             parent_path: fastn_ds::Path::new("/"),
         };
 
-        let resp = fastn_core::package::package_doc::read_ftd(req_config, &main, "/", false, false)
-            .await?;
+        return match fastn_core::package::package_doc::read_ftd(
+            req_config, &main, "/", false, false,
+        )
+        .await
+        {
+            Ok(resp) => {
+                // [SUCCESS] logging
+                req.log(
+                    fastn_core::log::EventKind::Auth(fastn_core::log::AuthEvent::Login),
+                    fastn_core::log::EntityKind::Myself,
```

Comment:
Since all auth related log will be done on myself, this Myself is useless. Remove it. 

Also I would prefer hard coding the event names, eg "login" instead of `fastn_core::log::EventKind::Auth(fastn_core::log::AuthEvent::Login)`, we will never be doing a match on this in Rust, this is *ONLY* going to be used in dashboard etc, and it would be currently a torture to find out what do put in where clause in my SQL query looking at this code.